cleanup.ts

```diff
@@ -35,8 +33,17 @@ const cleanupPasses: CleanupPass[] =
  *
  * To add a cleanup step, just add it to the array of passes above.
  */
-export async function cleanup(element: ElementRepo): Promise<void> {
-  for (const step of cleanupPasses) {
-    await step(element);
+export async function cleanup(
+    element: ElementRepo, config: CleanupConfig): Promise<void> {
+  for (const step of getPasses()) {
+    const stepConfig = config[step.name] || {};
+    if (!step.runsByDefault) {
```

Comment:
We'll need to have something here to enable non-default passes. Command line flag and/or config setting.